nixosTest: remove `machine` syntax sugar (#165035)
NixOS/nixpkgs: nixos/doc/manual/development/writing-nixos-tests.section.md

The attribute `testScript` is a bit of Python code that executes the
test (described below). During the test, it will start one or more

> virtual machines, the configuration of which is described by
> the attribute `nodes` (if you need multiple machines).

Reviewer: ```suggestion
virtual machines, the configuration of which are described by
the attribute `nodes`.
```
